AdyenDropIn/Session/DropInSession.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+public final class DropInSession {
+    internal weak var session: SessionProtocol?
+    
+    public init(session: SessionProtocol) {
+        self.session = session
+    }
+    
+    public func didFail(with error: Error, from dropInComponent: DropInComponent) {
```

Comment:
Why would someone call `didFail` on `DropInSession` ?
It could be `internal`, right?

Reply:
its needed by the `DropInComponentDelegate` protocol, so has to be public :-)